multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -56,6 +69,37 @@ coreos:
         [Install]
         WantedBy=multi-user.target
 
+{{ if eq .ContainerRuntime "rkt" }}
+    - name: rkt-api.service
+      enable: true
+      content: |
+        [Service]
+        ExecStart=/usr/bin/rkt api-service
+        Restart=always
+        RestartSec=10
+		[Install]
+        RequiredBy=kubelet.service
+        Before=kubelet.service
+        WantedBy=multi-user.target
```

Comment:
nix `WantedBy` line. This means "this program should be running if we made it to multi-user mode" which is not what we want to say.